docs/getting-started/create-comit-app.md

```diff
@@ -4,6 +4,286 @@ title: Getting Started
+- Create two HD wallet,
+- Funds each wallet with 10 Bitcoin,
+- Run `generate` every second to mine a new block.
+
+`Starting two cnds...✓`: Starts 2 cnd (COMIT Network Daemon) in docker containers.
+
+`Writing configuration in env file...✓`: Write useful information to `~/.create-comit-app/env`
+
+Let's check out the result:
+
+```
+▶ docker ps
+CONTAINER ID        IMAGE                             COMMAND                  CREATED             STATUS              PORTS                                                                                                  NAMES
+207bdd00d103        comitnetwork/cnd:0.4.0            "tini -- cnd --confi…"   26 minutes ago      Up 26 minutes       8000/tcp, 9939/tcp, 0.0.0.0:58249->8080/tcp                                                            cnd_1
+e72b5910e530        comitnetwork/cnd:0.4.0            "tini -- cnd --confi…"   26 minutes ago      Up 26 minutes       8000/tcp, 9939/tcp, 0.0.0.0:58248->8080/tcp                                                            cnd_0
+7c9855e6f3c0        coblox/bitcoin-core:0.17.0        "/usr/bin/bitcoind -…"   26 minutes ago      Up 26 minutes       8332-8333/tcp, 18332-18333/tcp, 0.0.0.0:58234->18443/tcp, 0.0.0.0:58233->18444/tcp                     bitcoin
+66a52568666d        coblox/parity-poa:v2.5.9-stable   "/bin/parity"            26 minutes ago      Up 26 minutes       5001/tcp, 8080/tcp, 8082-8083/tcp, 8180/tcp, 8546/tcp, 30303/tcp, 30303/udp, 0.0.0.0:58176->8545/tcp   ethereum
+```
+Indeed we have bitcoind, parity and cnds running.
+
+Let's have a look at this `env` file:
+
+```
+▶ cat ~/.create-comit-app/env  
+BITCOIN_HD_KEY_0=tprv8ZgxMBicQKsPf8tkuTQ3eftSjgnE17mEaoNbDz8r7N9TQdVJujs5yaoYpPR3HqetuD8g5hV3Ziqby4x6KB6yTcUyXBLDpqYv6we3etu9g5t
+BITCOIN_HD_KEY_1=tprv8ZgxMBicQKsPfCW2nYxkfxMnb49PyTYFCF7sdkHdNWBgm89dPii9vtZXfd6dJVwi95NCq9DpoAwLioWvdzVTtmGt2MLR39R9giGscJBsF9H
+BITCOIN_P2P_URI=127.0.0.1:58233
+ERC20_CONTRACT_ADDRESS=0x62d69f6867a0a084c6d313943dc22023bc263691
+ETHEREUM_KEY_0=77914ef506f41277bd62c9296143c68bbb8006d7c66619af42a626aced947654
+ETHEREUM_KEY_1=f8ce18c1bf5c586ef2f89ed04d0296c49bf915483d2866decddd487dc83488c9
+ETHEREUM_NODE_HTTP_URL=http://127.0.0.1:58176
+HTTP_URL_CND_0=http://127.0.0.1:58248
+HTTP_URL_CND_1=http://127.0.0.1:58249
+```
+
+We have here the private keys of the funded accounts on Ethereum and Bitcoin and the endpoint to the various services.
+This can be used to be imported in my-app so that you can test it with funded accounts.
+
+To stop the environment, just hit `CTRL+C` and wait few seconds:
+
+```
+▶ yarn run start-env
+yarn run v1.21.1
+$ comit-scripts start-env
+Creating Docker network (create-comit-app)...✓
+Starting Ethereum node...✓
+Starting Bitcoin node...✓
+Starting two cnds...✓
+Writing configuration in env file...✓
+🎉 Environment is ready, time to create a COMIT app!
+✓
+^C🧹 Cleaning up...
+✓
+```
+
+## Run a demo
+
+Let's run one of the demo to see how an atomic swap run looks like:
```

Comment:
```suggestion
`create-comit-app` provides several demo applications that allow you to do an atomic swap between . different assets.
Let's run one of the demos to see how an atomic swap run looks like:
```

Reply:
It's already mentioned at L110:
```
- `./demos` contains projects which are catered for workshops demonstration. They do not have a conventional structure but have verbose output.
```

Reply:
Yes, but given that this is its own section it would not hurt to repeat it.